### PR TITLE
WT-7084 Fix assert calls in test_base01

### PR DESCRIPTION
**Pull request by baiwfg2**
Hi. It's a minor test bug. Please have a check.

### CODE REVIEW

**Comment by lukech** [comment]
autobuild: Can one of the admins verify this patch?

**Comment by baiwfg2** [comment]
@lukech Hi, Luke. It seems that external developers are unauthorized ?

**Comment by lukech** [comment]
Hi @baiwfg2, thanks for raising the PR.
To be able to get your change reviewed and merged, could you take a look at and sign our [contributor agreement](https://www.mongodb.com/legal/contributor-agreement)?

**Comment by baiwfg2** [comment]
Thanks for reminding. Done.

**Comment by lukech** [comment]
Thanks @baiwfg2. We are still going through some internal process to confirm your signing. Hope you could understand.

File: test/suite/test_base01.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def test_error(self):
                 gotException = True
                 self.pr('got expected exception: ' + str(e))
                 self.assertTrue(str(e).find('nvalid argument') >= 0)
-        self.assertTrue(gotException, 'expected exception')
+        self.assertTrue(gotException)
```
Comment (ddanderson) [inline test/suite/test_base01.py:69]: `assertTrue` actually takes an optional second argument (see https://docs.python.org/3/library/unittest.html), that was the intention here.  To make it clearer, the code could be changed to:
```
  self.assertTrue(gotException, msg='expected exception')
```

**Review by ddanderson — COMMENTED** [review]
Hi @baiwfg2, thanks for the pull request.  I suggested a small modification to this change.

~~Also, we are waiting on your signing the contributor agreement.  We can accept contributions without that.~~

**Comment by ddanderson** [comment]
@baiwfg2, Oh I just saw that you commented that you signed it.  Then I'm waiting on our end to confirm it.  Thanks!

File: test/suite/test_base01.py (hunk 1 above)
Comment (baiwfg2) [reply test/suite/test_base01.py]: I see.

File: src/log/log.c [hunk 2]
```diff
@@ -2687,7 +2687,7 @@ __log_write_internal(WT_SESSION_IMPL *session, WT_ITEM *record, WT_LSN *lsnp, ui
             __wt_log_slot_free(session, myslot.slot);
     } else if (force) {
         /*
-         * If we are going to wait for this slot to get written, signal the wrlsn thread.
+         * If we are going to wait for this slot to get written, signal the log server thread.
```
Comment (baiwfg2) [inline src/log/log.c:2690]: Years ago the code below this comment was :
```
if (conn->log_wrlsn_cond != NULL) {
	WT_ERR(__wt_cond_signal(session, conn->log_wrlsn_cond));
	__wt_yield();
}
```
by Susan. I guess the comment is forgotten to be fixed. @sueloverso Am I right ?

**Comment by lukech** [comment]
@baiwfg2 We have confirmed your signing of the contributor agreement. Thanks for your patience. 
@ddanderson Are you happy with the change in commit 5772300?

**Comment by baiwfg2** [comment]
@lukech 😄 
BTW, 
1. When I have multiple PR ideas (related or unrelated), should I create one Jira issue or multiple ? 
2. One Jira issue fits with one commit ?
3. Create the Jira even if the fix is minor ?

**Comment by lukech** [comment]
@baiwfg2 
> When I have multiple PR ideas (related or unrelated), should I create one Jira issue or multiple ?

Related ideas can be put into the same Jira issue. 

> One Jira issue fits with one commit ?

Yes, we adopt the "one-ticket-one-commit" policy in terms of commit history. When merging a PR the "squash and merge" option would be chosen so that multiple commits (e.g. some to address review comments) of the PR could be merged into a single one. 

> Create the Jira even if the fix is minor ?

Correct, any change to be merged into the codebase needs a Jira issue.

File: src/log/log.c (hunk 2 above)
Comment (ddanderson) [reply src/log/log.c:2690]: Normally we would have a separate pull request to cover comment updates in another area of the code.  But I've already taken the trouble to verify this, so we'll include it.

**Review by ddanderson — COMMENTED** [review]
LGTM.

**Comment by ddanderson** [comment]
Thanks @baiwfg2, for the improvements!

**Pull request merged**